tests/test_apt.py

```diff
@@ -0,0 +1,57 @@
+'''Define unit tests for parsing APT templates with pytest.
+
+Authors
+-------
+    - Lauren Chambers
+
+Use
+---
+    Ensure you have pytest installed. Then, simply run pytest in any
+    parent directory of nircam_simulator/tests/:
+    >>> pytest
+'''
+
+import os
+
+from nircam_simulator.scripts import write_observationlist, yaml_generator, utils
+from nircam_simulator.scripts.get_catalog import get_all_catalogs
+
+INSTRUMENTS = ['NIRCam']
+PROPOSAL_ID = '1111'
+
+TESTS_DIR = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
+
+
+def test_APT():
```

Comment:
Can you please modify the name of the test function to be a bit more descriptive?

This test is missing an assert statement. I am actually not sure what happens if you run pytest on this?
Please add appropriate assert statements.

Reply:
Up until this point, I've really just been running pytest to ensure that all of the code was executed without throwing any errors. I'm not entirely sure what kinds of assert statements are useful here (please let me know if you have any ideas?).

For now at least, I added in a case that will compare the observationlist.yaml file that is created by the simulator to a reference file, if one is present in the output directory.